src/OpenTelemetry.AspNetCore/OpenTelemetry.AspNetCore.csproj

```diff
@@ -0,0 +1,15 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <Description>OpenTelemetry ASP.NET Core metapackage.</Description>
```

Comment:
Should we use the term "meta" package here? this is not a metapackage as this contains code.

Reply:
updated!